packages/node-core/test/schemas/schemaDiff.json

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
Do you think this is holding too much information?

We're redefining a schema here with all the details on the models and fields. I think a better, simpler option is to just have this data type have the minimal information to changes in schemas then use the new graphql schema as the source of truth with all the details like nullable etc.